packages/client/src/Components/ChatChannel.tsx

```diff
@@ -66,27 +69,62 @@ const ChatChannel: React.FC<{ channelId: string }> = ({ channelId }) => {
   // TODO: we have some wrong typing here.  The messages for this channel
   // will all be chat messages plus turn markers.  But, that doesn't match
   // what data is stored in the the channels dictionary
-  const chatMessages = state.channels[channelId].messages as any as ChatMessage[]
+  const messages = state.channels[channelId].messages as any
+  
+  const onRead = (detail: MessageCustom): void => {
+    const newState = isRead.map((state, id) => {
```

Comment:
@lilitkarapetyan - when a message is marked as
 `read`, don't we have to store something in `expired-storage` - so it remains `read` after browser refresh?